index.html

```diff
@@ -6,9 +6,37 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>portfolio</title>
 
+    <!-- Bootstrap icons link -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.2/font/bootstrap-icons.css"
+    />
     <link rel="stylesheet" href="style.css" />
   </head>
   <body>
-    <h1 class="header">"Hello!"</h1>
+    <header class="header">
+      <nav class="navbar">
+        <div class="logo">BZ.</div>
+        <i class="bi bi-list"></i>

```

Comment:
- Please try to increase the thickness of the hamburger menu so as it looks like that in the figma, presently yours looks very slim and small

##### Yours
![Screenshot from 2022-05-25 22-10-27](https://user-images.githubusercontent.com/58771507/170367971-833ea4a8-2082-4f54-a55c-2cc384ba8e85.png)

##### Figma
![Screenshot from 2022-05-25 22-11-27](https://user-images.githubusercontent.com/58771507/170368108-39cf1aa0-2e34-4b0e-9a79-07a1fd3ed6cd.png)